### ISSUE TITLE
Simulator for Robot

### ISSUE THREAD

**Issue by TibiGG** [jringert/gendev.bot#1]
I was wondering whether there is any tool already made to simulate the behaviour of the robot. Thank you!

**Comment by jringert** [comment]
No, we don't have a tool for that, sorry. However, the separation of the execution via a socket- or HTTP-based interface should allow for simulating it in almost any framework (you could even hack one in JavaScript, I guess). 

We have reworked the communication part a bit to use sockets instead of HTTP (faster communication for the ESP32/8266). If you are interested in the robot, please let me know, some of the code might be outdated in the repository, e.g., the symbolic controller execution JAR file needs replacing with a later one from the current Spectra version.

**Comment by TibiGG** [comment]
Ok, thank you for the quick reply.

Yes, I am interested, and most probably will build the robot. Thank you for letting me know of the outdated jar file. Still, I found that I could run the RobotRunner.java file without any other problem than the connection to a robot, so I will most probably see the issues when I will connect to a real or simulated robot.
